templates/paragraph/paragraph--localgov-call-out-box.html.twig

```diff
@@ -1,16 +1,27 @@
+{%
+  set classes = [
+    'call-out-box',
+    colour_theme ? 'call-out-box--' ~ colour_theme|clean_class,
+    'row',
+    'mx-1'
+  ]
+%}
 
-<div class="call-out-box row mx-1">
-  <div class="col-12 col-md-6 order-md-2 p-0" />
-    {% if background_url is defined and background_url %}
-      <img src="{{ background_url }}" />
-    {% endif %}
-  </div>
+<div{{ attributes.addClass(classes) }}>
```

Comment:
Can you add in the classes 'call-out-box row mx-1' to this like so: <div{{ attributes.addClass(classes, 'call-out-box',  'row', 'mx-1') }}>

Reply:
Not sure I understand, those classes are already included in the `classes` array at the top of the file?

Reply:
ah ok - didnt see that i thought the 'classes' were being passed in from some user panel in admin